content/maintainers-guide/reviewing-pull-requests.md

```diff
@@ -1,156 +1,215 @@
+| The PR is waiting for the author to address feedback                                       | The PR's author  |   <label type="needs">Waiting for author</label>   |                _(no label)_                 |
+| The PR must be rebased<br><small>(because of merge conflicts or wrong base branch)</small> | The PR's author  |   <label type="needs">Waiting for rebase</label>   |                _(no label)_                 |
+| The PR introduced functional changes                                                       | Product Managers |     <label type="needs">Waiting for PM</label>     |      <label type="check">PM ✓</label>       |
+| The PR introduced visual changes                                                           |   UX Designers   |     <label type="needs">Waiting for UX</label>     |      <label type="check">UX ✓</label>       |
+| The PR introduced wording changes                                                          | Wording Managers |  <label type="needs">Waiting for Wording</label>   |    <label type="check">Wording ✓</label>    |
+| The PR is ready for QA verification                                                        | Software Testers |     <label type="needs">Waiting for QA</label>     |      <label type="check">QA ✓</label>       |
+| The PR needs to be documented in DevDocs                                                   |   Maintainers    | <label type="needsdoc">Needs documentation</label> | <label type="check">Documentation ✓</label> |
+
+### Meta labels
+
+The following labels provide metadata and are essentially informative:
+
+| Label                                          | Meaning                                                      |
+|:-----------------------------------------------|:-------------------------------------------------------------|
+| <label type="type">Bug fix</label>             | This PR fixes a bug                                          |
+| <label type="type">Feature</label>             | This PR introduces a new feature                             |
+| <label type="type">Improvement</label>         | This PR is an improvement (e.g. performance)                 |
+| <label type="type">Refactoring</label>         | This PR is a refactoring                                     |
+| <label type="branch">1.7.8.x</label>           | Destination branch (e.g. 1.7.8.x, develop, ...)              |
+| <label type="bcbreak">BC break</label>         | This PR includes breaking changes                            |
+| <label type="keyfeature">Key feature</label>   | This PR includes a feature to be highlighted in this release |
+| <label type="wip">WIP</label>                  | This PR is a work in progress                                |
+| <label type="e2e">E2E</label>                  | This PR is about end-to-end automated tests                  |
```

Comment:
I think there is no such label anymore, it's `TE` and `E2E Tests`

Reply:
I see some open PRs with that label: https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Aopen+is%3Apr+label%3A%22E2E+Tests%22

Question: is the "TE" label really needed?

Reply:
E2E might be an old label

Reply:
> 

- `E2E TESTS` is the one used for UI tests.
- `TE` is a pull request category for all tests. and we used it to run lint on UI tests (we should use `E2E Tests` for that)
- `E2E` (without `TESTS`) is something i never used personally and should be deleted.

Reply:
`TE` is linked to the PR category (refacto, tests ...), if a PR has only modification on unit or integrations tests, it's not `E2E Tests` but the PR category is `TE`.